components/assetcard/index.css

```diff
@@ -13,52 +13,84 @@ governing permissions and limitations under the License.
-
-  --spectrum-assetcard-heading-text-color: var(--spectrum-global-color-gray-900);
-  --spectrum-assetcard-content-text-color: var(--spectrum-global-color-gray-700);
+  --spectrum-assetcard-border-color-hover: var(--spectrum-gray-500);
+  --spectrum-assetcard-border-color-down: var(--spectrum-gray-600);
+
+  --spectrum-assetcard-focus-ring-gap: 5px;
+  --spectrum-assetcard-focus-indicator-thickness: var(--spectrum-focus-indicator-thickness);
+
+  /* selection indicator - checkbox or number */
+  --spectrum-assetcard-selectionindicator-background-color-default: rgba(var(--spectrum-gray-100-rgb), 0.9);
+  --spectrum-assetcard-selectionindicator-size: var(--spectrum-card-selection-background-size);
+  --spectrum-assetcard-selectionindicator-border-radius: var(--spectrum-corner-radius-100);
+  --spectrum-assetcard-selectionindicator-offset-y: 4px;
+  --spectrum-assetcard-selectionindicator-blur: 6px;
+  --spectrum-assetcard-selectionindicator-color: var(--spectrum-white);
+  --spectrum-assetcard-selectionindicator-font-weight: var(--spectrum-bold-font-weight);
+  --spectrum-assetcard-selectionindicator-font-size: var(--spectrum-font-size-400);
+
+  /* title */
+  --spectrum-assetcard-title-text-color: var(--spectrum-gray-900);
+  --spectrum-assetcard-title-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-assetcard-title-font-weight: var(--spectrum-heading-sans-serif-font-weight);
+  --spectrum-assetcard-title-font-style: var(--spectrum-default-font-style);
+  --spectrum-assetcard-title-line-height: var(--spectrum-line-height-100);
+  --spectrum-assetcard-title-letter-spacing: 0;
+
+  /* header content */
+  --spectrum-assetcard-header-content-text-color: var(--spectrum-gray-900);
+  --spectrum-assetcard-header-content-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-assetcard-header-content-font-weight: var(--spectrum-body-sans-serif-font-weight);
+  --spectrum-assetcard-header-content-font-style: var(--spectrum-default-font-style);
+  --spectrum-assetcard-header-content-line-height: var(--spectrum-line-height-200);
+  --spectrum-assetcard-header-content-letter-spacing: 0;
+
+  /* content */
+  --spectrum-assetcard-content-text-color: var(--spectrum-gray-700);
+  --spectrum-assetcard-content-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-assetcard-content-font-weight: var(--spectrum-body-sans-serif-font-weight);
+  --spectrum-assetcard-content-font-style: var(--spectrum-default-font-style);
+  --spectrum-assetcard-content-line-height: var(--spectrum-line-height-200);
+  --spectrum-assetcard-content-letter-spacing: 0;
+
+  --spectrum-assetcard-content-margin-block-start: var(--spectrum-spacing-75);
+}
 
+.spectrum-AssetCard {
   /* contain selection indicator */
   position: relative;
 
   display: flex;
   flex-direction: column;
 
-  inline-size: var(--spectrum-assetcard-asset-size);
+  inline-size: var(--mod-assetcard-asset-size, var(--spectrum-assetcard-asset-size));
```

Comment:
I noticed a token for minimum width - should we account for that here too? `var(--spectrum-card-minimum-width)`

Reply:
The `--spectrum-card-minimum-width` token is used in the separate card component. I think It would make sense to move to that but I was not sure if that's with in scope of this migration.